vpn_slice/mac.py

```diff
@@ -42,6 +42,8 @@ def _ifconfig(self, *args):
 
     def add_route(self, destination, *, via=None, dev=None, src=None, mtu=None):
         args = ['add']
```

Comment:
Perhaps we can make this a bit less verbose on the Python side, and a bit more explicit on the `route` side, by using something like this instead?

```python
args = ['add', ('-inet6' if destination.version == 6 else '-inet')]
```

Reply:
Sure. I'll make that modification.

Reply:
:+1: :+1: :+1: